tests/benchmarks/test_registry.py

```diff
@@ -28,8 +28,7 @@ def add_parser_arguments(self):
         self._parser.add_argument(
             '--lower_bound',
             type=int,
-            default=0,
-            required=False,
+            required=True,
```

Comment:
Do we need to add a test for this? It's a little wired that we revise the test.

Reply:
The current test case can trigger and test it with this change.